### PR TITLE
realm: Add Sentry

### PR DESCRIPTION
**Pull request by backspace**
This adds a `/_boom` route to deliberately trigger an error and also reports when there’s an indexing error. Are there other errors that should be reported?

It took a lot of experimentation to get it working in the Fastboot environment, let me know if there’s a better way.

### CODE REVIEW

**Review by jurgenwerk — APPROVED** [review]

**Comment by jurgenwerk** [comment]
Wondering if we should report when indexing errors happens too?

**Comment by backspace** [comment]
ya definitely, I first just wanted to be able to see _any_ failure in Sentry 🤓

File: packages/realm-server/package.json [hunk 1]
```diff
@@ -91,5 +91,8 @@
   },
   "volta": {
     "extends": "../../package.json"
+  },
+  "dependencies": {
+    "@sentry/node": "^7.105.0"
```
Comment (backspace) [inline packages/realm-server/package.json]: Should this be in `devDependencies` like everything else? I’m confused why that’s where things are.

**Review by lukemelia — APPROVED** [review]

**Review by habdelra — APPROVED** [review]
i agree with this approach :+1:

**Pull request merged**